codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -239,6 +239,7 @@ void removeStream(DefaultStream stream) {
         private State state = IDLE;
         private short weight = DEFAULT_PRIORITY_WEIGHT;
         private DefaultStream parent;
+        // TODO(nathanmittler): We no longer need a map. LinkedList?
```

Comment:
We don't use the map for anything.  I checked, the `child(id)` method is only used by a test.  Seems like a different data structure is more appropriate.